clients/java/signalr/src/test/java/com/microsoft/signalr/WebSocketTransportTest.java

```diff
@@ -19,7 +19,7 @@
     @Test
     public void WebsocketThrowsIfItCantConnect() {
```

Comment:
nit: while you're here

```suggestion
    public void WebSocketThrowsIfItCantConnect() {
```